bindings_node/src/encoded_content.rs

```diff
@@ -0,0 +1,81 @@
+use napi::bindgen_prelude::Uint8Array;
+use napi_derive::napi;
+use std::{collections::HashMap, ops::Deref};
+use xmtp_proto::xmtp::mls::message_contents::{ContentTypeId, EncodedContent};
+
+#[derive(Clone)]
+#[napi(object)]
+pub struct NapiContentTypeId {
+  pub authority_id: String,
+  pub type_id: String,
+  pub version_major: u32,
+  pub version_minor: u32,
+}
+
+impl From<ContentTypeId> for NapiContentTypeId {
+  fn from(content_type_id: ContentTypeId) -> NapiContentTypeId {
+    NapiContentTypeId {
+      authority_id: content_type_id.authority_id,
+      type_id: content_type_id.type_id,
+      version_major: content_type_id.version_major,
+      version_minor: content_type_id.version_minor,
+    }
+  }
+}
+
+impl Into<ContentTypeId> for NapiContentTypeId {
```

Comment:
do i need an `Into` trait when there's a `From` trait?

Reply:
That automatically gets generated for you based on the `From`